sqlstore/src/it/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateProjectDaoIT.java

```diff
@@ -69,4 +72,17 @@ public void testGetUsage() throws Exception {
     testGetUsage(HibernateProjectDao::getUsage, 1L, 21L);
   }
 
+
+  @Test
+  public void testAssays() throws IOException {
+    String code = "TEST1";
+    Set<Assay> assays = projectDAO.getByCode(code).getAssays();
+    assertNotNull(assays);
+    boolean checker = false;
+    if (assays.size() > 0) {
+      checker = true;
+    }
+    assertTrue(checker);

```

Comment:
* could be reduced to `assertTrue(assays.size() > 0)`
* better to test specific value. There should be 2 assays, so `assertEquals(2, assays.size())`
* even better to test that the correct assays are added (assay IDs 1 and 2). Note you can't depend on these being returned in any particular order, so can do something like `assertTrue(assays.stream().anyMatch(...))`